definitions/linux.yaml

```diff
@@ -38,6 +38,16 @@ sources:
 labels: [Configuration Files]
 supported_os: [Linux]
 ---
+name: LinuxScheduleFiles
+doc: All Linux job scheduling files.
+sources:
+- type: ARTIFACT
+  attributes:
+    names: [AnacronFiles, LinuxCronTabs, LinuxAtJobs]
+  returned_types: [StatEntry]
```

Comment:
Lets drop returned_types, unless there's a strong reason to keep it.  This is a GRR-ism I'd like to keep out of this repo as much as possible.

Otherwise, looks great, thanks!